code/datums/greyscale/config_types/greyscale_configs/greyscale_clothes.dm

```diff
@@ -360,8 +360,8 @@
 
 /datum/greyscale_config/labcoat
 	name = "Labcoat"
-	icon_file = 'icons/mob/clothing/suits/labcoat.dmi'
-	json_config = 'code/datums/greyscale/json_configs/labcoat_worn.json'
+	icon_file = 'icons/obj/clothing/suits/labcoat.dmi'
+	json_config = 'code/datums/greyscale/json_configs/labcoat.json'
```

Comment:
We should be able to cut down to only a single json_config (that was the hope with having different icon-files)

If you're willing to do that:
- Replace your current labcoat.json with this one - it has the missing Back component-states.
[labcoat.json](https://github.com/tgstation/tgstation/files/13236195/labcoat.json)

 - Then, remove the json_config define from greyscale_config/labcoat/worn (below this line) - it will inherit the json_config from here, and thus both the object and worn sprites can use the exact same set of code.

_Adding those blank 'back' states also fixed the only issue this would have caused too...
so really there were two solutions: add new icons and have them share a json, or set it to the other intended json. You basically did both fixes at once LMAO_

Reply:
Did that and removed worn file